src/java/com/zimbra/oauth/token/handlers/impl/NextCloudTokenHandler.java

```diff
@@ -70,19 +70,17 @@ public static String refreshAccessToken(Account acct, String client) throws Serv
         final OAuthInfo oauthInfo = new OAuthInfo(new HashMap<String, String>());
         final String refreshToken = OAuth2DataSource.getRefreshToken(dataSource);
 
-        String value = com.zimbra.oauth.token.utilities.LdapOauthHelper.getFirstConfig(Provisioning.A_zimbraOAuthConsumerCredentials,
-                client, acct) ;
-        String indVa [] = value.split(":");
+        String [] values = LdapOauthHelper.loadConfiguration(acct, Provisioning.A_zimbraOAuthConsumerCredentials,
```

Comment:
This won't work. If you use loadConfiguration you'll get all configs - including the other clients.